src/controllers/UserController.js

```diff
@@ -39,4 +40,32 @@ export default class UserController {
       return Responses.send(res);
     });
   }
+
+  /**
+   * @method
+   * @description Implements signin endpoint
+   * @static
+   * @param {object} req - Request object
+   * @param {object} res - Response object
+   * @returns {object} JSON response
+   * @memberof UserController
+   */
+  static async signin(req, res) {
+    const { email, password } = req.body;
+    const authErrorMessage = 'Incorrect email or password.';
```

Comment:
You should validate outside the controller class

Reply:
ok